### ISSUE TITLE
Save Our Ship 2 integration via Visitor Portal

### ISSUE THREAD

**Issue by TaylorVoth** [OrionFive/Hospitality#468]
1 word: Space Hotel. 

A visitor portal or something added to allow Hospitality summoned visitors to warp into and out of my spaceship and stay as normal.

**Comment by OrionFive** [comment]
Sure, go ahead, let me know when it's done :P

**Comment by xFilosofx** [comment]
Yeah, this be very immersive integration. On late game this will be fun.
